web/packages/teleterm/src/ui/services/connectionTracker/connectionTrackerService.ts

```diff
@@ -256,6 +256,7 @@ export class ConnectionTrackerService extends ImmutableStore<ConnectionTrackerSt
               const newItem = createGatewayConnection(doc);
               draft.connections.push(newItem);
             } else {
+              // In case the document changes, update the gateway title.
```

Comment:
I don't think this is specific enough. IIRC this was to specifically address a case where over time we've decided to change the title for db gateways and we wanted this change to be reflected in already created connections, right?

Reply:
Yeah, I mentioned it now it the comment.